mod-source-record-manager-server/src/main/java/org/folio/rest/impl/InitAPIImpl.java

```diff
@@ -43,13 +43,13 @@ public class InitAPIImpl implements InitAPI {
   @Value("${srm.kafka.RawMarcChunkConsumer.instancesNumber:1}")
   private int rawMarcChunkConsumerInstancesNumber;
 
-  @Value("${srm.kafka.StoredMarcChunkConsumer.instancesNumber:3}")
+  @Value("${srm.kafka.StoredMarcChunkConsumer.instancesNumber:1}")
```

Comment:
@SerhiiNosko  I think we need to mention these changes in release notes. What do you think?

Reply:
Updating page with recommended params for deployment to mention about this, will post link after update's complete, thanks